pages/settings/sessions.vue

```diff
@@ -44,14 +63,55 @@ definePageMeta({
 })
 
 useHead({
-  title: 'Sessions - Modrinth',
+  title: () => `${formatMessage(messages.sessionsTitle)} - Modrinth'`,
 })
 
 const data = useNuxtApp()
 const { data: sessions, refresh } = await useAsyncData('session/list', () =>
   useBaseFetch('session/list')
 )
 
+const vintl = useVIntl()
+const { formatMessage } = vintl
```

Comment:
```suggestion
const { formatMessage } = useVIntl()
```
I don't see any uses of `vintl`, so this variable declaration is unnecessary and you can destructure right away.